pyanaconda/ui/cockpit/anaconda-webui/src/InstallationDestination.jsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
```

Comment:
I'm thinking we should discuss what license we want to use for the Web UI files - traditionally we have used GPL 2+ for almost everything in Anaconda with LGPL 2+ in just a couple places, mostly some utility scripts.

Reply:
Indeed. wednesday topic it is.